packages/block-library/src/subhead/index.js

```diff
@@ -2,12 +2,12 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { subtitle as icon } from '@wordpress/icons';
```

Comment:
I think we retired the subheading block. Moreso perhaps "subtitle" isn't the best name as I think of that as video subtitles, and it's entirely possible a video block lands in the future with the ability to toggle subtitles.

Reply:
It's retired yes but it's still in the code with its own icon. So open for suggestions about the naming here (check the PR description for questions about the naming)